app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -127,8 +135,9 @@ import javax.inject.Singleton
   qualifiers = "port-xxhdpi"
 )
 class ExplorationActivityTest {
-  @get:Rule
-  val accessibilityTestRule = AccessibilityTestRule()
```

Comment:
Why was this removed? We need this check to make sure the app is accessible by all and tests pass on it.

Reply:
done